configure.ac

```diff
@@ -14,7 +14,7 @@ AM_PROG_AS
 AC_CHECK_HEADERS_ONCE([iconv.h])
 
 # Checks for library functions.
-AC_CHECK_FUNCS([strdup strndup])
+AC_CHECK_FUNCS([strdup strndup clock_gettime])
```

Comment:
The check result for `clock_gettime` is never used *(`$ac_cv_func_clock_gettime` during configure or `HAVE_CLOCK_GETTIME` in the C preprocessor)*. If the result is not used, the check isn’t particularly useful.

Reply:
Removed.